app/views/finders/_spelling_suggestion.html.erb

```diff
@@ -1,10 +1,16 @@
+<% # we want an empty string if there are no suggestions on page load %> 
+<% _spelling_suggestion = '' %>
 <% if @spelling_suggestion_presenter.suggestions.any? %>
 <p class="govuk-body">Did you mean
   <% @spelling_suggestion_presenter.suggestions.each do |suggestion| %>
     <%= link_to suggestion[:keywords], suggestion[:link],
       class: "govuk-link govuk-!-font-weight-bold",
       :data => suggestion[:data_attributes]
     %>
+    <% _spelling_suggestion << suggestion[:keywords] %>
```

Comment:
`<<` for strings is concatenation.  Because we only have one suggestion and `_spelling_suggestion` starts as `''`, this *really* means `_spelling_suggestion = suggestion[:keywords]`.  Would it be clearer to say that directly?

Reply:
yep, thought about that first. will update it